src/atoms/Link/Link.web.js

```diff
@@ -1,3 +1,50 @@
-const Link = () => null;
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import Text from '../Text';
+import { getColor } from '../../_helpers/theme';
+
+const Anchor = styled(Text)`
+  color: ${(props) => getColor(props.theme, 'primary.800')};
+  text-decoration: none;
+  cursor: pointer;
+  &:hover {
+    text-decoration: underline;
+    color: ${(props) => getColor(props.theme, 'primary.800')};
+  }
+  &:focus {
+    outline: none;
+    text-decoration: underline;
+    color: ${(props) => getColor(props.theme, 'primary.900')};
+  }
+  &:active {
+    outline: none;
+    text-decoration: underline;
+    color: ${(props) => getColor(props.theme, 'primary.700')};
+  }
+  &:visited {
+    outline: none;
+    color: ${(props) => getColor(props.theme, 'primary.700')};
+  }
+`;
+
+const Link = ({ size, children, onClick }) => {
+  return (
+    <Anchor as="a" size={size} onClick={onClick}>
+      {children}
+    </Anchor>
+  );
+};
+
+Link.propTypes = {
```

Comment:
Test ID is missing. Krishna is working on test id too. Maybe you can ask him how's he doing it. Rest of it looks good!

Reply:
Do we need a Test ID for this component?

Reply:
@susheelachoudry Yes.